Cargo.toml

```diff
@@ -1,6 +1,7 @@
 [workspace]
 members = [
     "cairo",
+    "crypto",
```

Comment:
Actually I don't know if `crypto` could be a misleading/too general name...

Reply:
I'll split it up as David suggested.